platforms/quorum/configuration/roles/create/k8s_secrets/tasks/main.yaml

```diff
@@ -0,0 +1,60 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+#############################################################################################
+# This role creates secrets 
+#############################################################################################
+
+# This task checks if Ambassador credentials exist already
+- name: Check Ambassador cred exists
```

Comment:
Move this check at the end, just before the task where it is created if not present.